### PR TITLE
fix (css): fixed scrolling

### PR DESCRIPTION
**Pull request by JuliaV95**

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@ body, html, #root {
     background-color: white;
     display: flex;
     flex-direction: column;
+    overflow: scroll;
```
Comment (dryabinin-actionengine) [inline src/scss/main.scss:29]: You don't need this scroll.

**Comment by dryabinin-actionengine** [comment]
You should cover such cases as long description of todo:

File: src/scss/main.scss (hunk 1 above)
Comment (JuliaV95) [reply src/scss/main.scss:29]: shit man god damn!

**Comment by dryabinin-actionengine** [comment]
1. Great now it looks better
2. Also scroll bar shouldn't be always displayed, just display it only if content height is more then container height: `overflow-y: auto`

**Pull request merged**